### DISCUSSION TITLE
Instructions for setting up a contact me page for a Quarto website

### DISCUSSION THREAD

**Discussion by mcewenkhundi** [quarto-dev/quarto-cli#784] [Q&A] [answered]
Hi I am looking for instructions on how I can set up a contact page of a Quarto website. I have searched the official documentation but couldn't find any mention of a contact page.

**Comment by dragonstyle** [comment, marked as answer]
I think the closest thing we have is 'About' pages:

https://quarto.org/docs/websites/website-about.html

**Reply by mcewenkhundi** [reply]
Noted
